core/app/[locale]/(default)/product/[slug]/page-data.ts

```diff
@@ -53,18 +269,18 @@ const ProductPageQuery = graphql(
     }
   `,
   [
-    BreadcrumbsFragment,
-    GalleryFragment,
-    DetailsFragment,
-    ProductItemFragment,
-    DescriptionFragment,
-    WarrantyFragment,
+    ProductDetailsFragment,
+    ProductViewedFragment,
+    ProductSchemaFragment,
+    // DetailsFragment,
+    // DescriptionFragment,
+    // WarrantyFragment,
```

Comment:
Are we going to be revisiting these later?

Reply:
Yes, didn't want to delete because I might need it for the next PRs. Good call tho, I should've left a comment.